nixos/modules/config/system-path.nix

```diff
@@ -135,6 +135,8 @@ in
       # outputs TODO: note that the tools will often not be linked by default
       postBuild =
         ''
+          find $out/bin -maxdepth 1 -name ".*" -type l -delete
```

Comment:
We need a comment for what this does, and could the match to the wrapped files not be on `.*`.
I'm doubtful that hidden files should be in the system path, but this doesn't actually just grab wrapper files.

Reply:
Yeah good call. ```.*-wrapped``` works